bedrock/firefox/templates/firefox/firstrun/email-first-experiment/index-a.html

```diff
@@ -0,0 +1,107 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
These two templates are copies of firefox_quantum.html with the necessary changes.

Reply:
Instead of copying the entire template for each variation, I suggest you just extend `firefox/firstrun/firstrun_quantum.html`.

```
{% extends "firefox/firstrun/firstrun_quantum.html" %}
```
In `firstrun_quantum.html` you can then put jinja blocks around each piece of HTML you wish to change in the test:

{% block iframe %}
  // original iframe html
{% endblock %}

In the variations you can then override the content using:

```
{% block iframe %}
  <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}signin?utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source=firstrun-login&amp;utm_content=fx-{{ version }}&amp;entrypoint=firstrun&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
{% endblock %}

Reply:
Done